source/features/discussion-links-on-repo-lists.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import issueIcon from 'octicon/issue-opened.svg';
+import pullRequestIcon from 'octicon/git-pull-request.svg';
+import observeElement from '../libs/simplified-element-observer';
```

Comment:
I'm not sure why this rule isn't active: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

First sort by distance: node_modules, then same directory, then `..` last

And then in each of these 3 groups you can sort the "from" keyword

```suggestion
import React from 'dom-chef';
import select from 'select-dom';
import issueIcon from 'octicon/issue-opened.svg';
import pullRequestIcon from 'octicon/git-pull-request.svg';
import features from '../libs/features';
import observeElement from '../libs/simplified-element-observer';
```